src/job/Job.ts

```diff
@@ -8,7 +8,7 @@ export type MomoJobStatus = WithoutId<JobEntity>;
 export interface JobDefinition {
   name: string;
   interval: string;
-  firstRunAfter: number;
+  firstRunAfter: number | string;
```

Comment:
One thing I wonder here: should we just save numbers to the job for human readable times?
E.g. A job and job entity have only numbers in `firstRunAfter` and `interval`. Then we do not need to keep parsing them. We parse once when validation/building a job.

Only minus point imo is that we will need a migration...

Reply:
Yes, I started thinking about the exact same point. It's annoying that we parse stuff several times. To avoid a migration and to keep readability of the db entities, I would suggest to save both - the interval and the parsed interval etc

I can try to implement that (but would rather do that in a separate PR)

Reply:
Hm, of course we can only avoid a migration if we make sure our code can handle job entities without parsed interval ... hmmm ... will need to look into that. Maybe it's actually nicer to run a migration - not sure

Reply:
That sounds reasonable

I was thinking about cases when multiple instances synch over one DB, but as long as the entity is backwards compatible it should be fine.